testing/test_aux.out

```diff
@@ -23,8 +23,8 @@ RESULT pass security.tls.v3
 RESULT pass security.x509
 dhcp requests 2 1
 01: ['01:brute:1']
-02: ['02:nmap:1', '02:udmi:2']
-03: ['03:nmap:1', '03:udmi:2']
+02: ['02:nmap:1']
```

Comment:
This seems to make the test fail. See:
https://travis-ci.org/arupiot/daq/jobs/554336934#L7140

Reply:
Travis will still fail on pull request builds, I'll have these fixed by the end of today.

if you look at the top of the log it's not picking up the encrypted `GCP_SERVICE_ACCOUNT` env var.

The last two Travis runs here:

https://github.com/InBrewJ/daq/commits/cloud_test_travis_config

pass, e.g. https://travis-ci.org/InBrewJ/daq/jobs/554418910

The report output here meshes with Trevor's fix to the report output when he was making the cloud udmi test work here:

https://github.com/grafnu/daq/blob/master/testing/test_aux.out

You changed the report output here to make Travis work when Travis wasn't successfully contacting our GCP account here:

https://github.com/arupiot/daq/commit/5a795f72bf5095d92e3c96481f85264203a61fa8

but I suspect it was a surface detail thing before we knew what we know now about Travis.

Reply:
It's the same issue these guys were having:

https://github.com/OHDSI/Achilles/issues/165

Reply:
@pisuke This PR should now be ready for review - see the passing Travis build under the screenshot below
(https://travis-ci.org/arupiot/daq/builds/554657702?utm_source=github_status&utm_medium=notification)